README.md

```diff
@@ -22,6 +22,10 @@ Provides a shortcode for displaying [academic calendar](http://calendar.ucf.edu/
 
 ## Changelog ##
 
+### 1.0.3 ###
+Enhancements:
+* Added github plugin URI meta to is can be installed via the Github Updater Plugin.
```

Comment:
Re-word this to something like "Added Github Updater Plugin URI meta to main plugin file", or just "Added support for installation via Github Updater"